### PR TITLE
CORE-10180: Escape string parameters when creating json to prevent possible injection attacks

### PR DESCRIPTION
**Pull request by anien**
- Escape string parameters
- Fix most of the kdoc parameters

### CODE REVIEW

**Comment by dickon** [comment]
Looks okay, probably needs some unit tests.

File: tools/plugins/initial-config/build.gradle [hunk 1]
```diff
@@ -21,6 +21,8 @@ dependencies {
     kapt "info.picocli:picocli:$picocliVersion"
 
     implementation 'net.corda:corda-config-schema'
+    implementation "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
```
Comment (ronanbrowne) [inline tools/plugins/initial-config/build.gradle]: Just double-checking you need

`implementation "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"`

as far as I see you are only using 

`com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
`

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
See dependency query

File: tools/plugins/initial-config/build.gradle (hunk 1 above)
Comment (anien) [reply tools/plugins/initial-config/build.gradle]: Ah, good point. I was trying few bits and didn't check which packages I need at the end. I'll double check.

**Comment by dickon** [comment]
LGTM

**Review by dickon — DISMISSED** [review]
LGTM

**Comment by anien** [comment]
Changing base branch shows some outstanding commits that shouldn't be part of this PR. I'll create a new one.

**Pull request closed without merging**